addons/mail/static/src/models/rtc_session/rtc_session.js

```diff
@@ -384,6 +379,13 @@ function factory(dependencies) {
         isMuted: attr({
             default: false,
         }),
+        /**
+         * Determine whether current session is either muted or deaf.
+         */
+        isMuteOrDeaf: attr({
+            compute: '_computeIsMuteOrDeaf',
+            default: false,
+        }),
```

Comment:
I think the modelling is off because the concept of `isMuted` is kept unchanged in this diff, but it should be considering all these changes.

Let me explain: `isMuteOrDeaf` seems introduced to determine user is unable to speak, whether it's from explicit mute action or deafen action. The latter case come from user being in a "deaf-mute" state. For me, this state **must** be `isMuted: true`.

So IMO the fields `isMuted` and `isDeaf` should be computed now.

The previous code made the quick assumption that user interaction to mute/deafen directly change his/her mute or deaf state. This is no longer true. There should be intermediary fields for them, like `isSelfMuted` and `isSelfDeafen` ("self" is there to tell the user made itself muted or deafen, which could be different from actually being muted or deafen). Feel free to propose another naming. These fields are nuanced so it's very important to properly document what they are and how they semantically differ.

Reply:
Sounds good, I'll have to make sure it remains consistent from both remote and local sessions mute/deaf states

Reply:
@alexkuhn Should this be in this PR too? it's adding a decent amount of changes for a fix (including an insert in python), see last commit.

Reply:
We should not make the rename in stable indeed.